### PR TITLE
Add new UI elements

### PR DESCRIPTION
**Pull request by xmasterpiecex**

### CODE REVIEW

File: src/components/gameScene/DrawerMain.tsx [hunk 1]
```diff
@@ -10,7 +10,9 @@ import {
 import Drawer from "@mui/joy/Drawer";
 import Menu from "@mui/icons-material/Menu";
 import { useState } from "react";
-import SoundSlide from "@app/components/gameScene/SoundSlide";
+import { useNavigate } from "react-router-dom";
```
Comment (uspiritwolf) [inline src/components/gameScene/DrawerMain.tsx]: Please remove unused import

File: src/pages/GamePage.tsx [hunk 2]
```diff
@@ -43,6 +47,7 @@ export default function GamePage() {
               );
             }
           })}
+          {amMaster && <EmptyPlayerPLace />}
```
Comment (uspiritwolf) [inline src/pages/GamePage.tsx]: Please correct the name of component, i.e. the word: PLace -> Place

**Review by uspiritwolf — CHANGES_REQUESTED** [review]

File: src/components/gameScene/DrawerMain.tsx (hunk 1 above)
Comment (xmasterpiecex) [reply src/components/gameScene/DrawerMain.tsx]: Got it.

File: src/pages/GamePage.tsx (hunk 2 above)
Comment (xmasterpiecex) [reply src/pages/GamePage.tsx]: Good point, i'm on it

**Review by uspiritwolf — APPROVED** [review]

**Pull request merged**